CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
-### 1.0.0 - 2019-01-19
+## Version 1.1.0 - 2019-02-07
```

Comment:
Check how https://github.com/ArkEcosystem/core/blob/develop/CHANGELOG.md formats and links the versions.